docs/dg/report.md

```diff
@@ -18,9 +18,9 @@ The following is a snapshot of the report:
 
 ## Report architecture
 
-![report architecture](../images/report-architecture.png)
```

Comment:
Line 15: For the report architecture, we haven't been able to get it to auto-render as puml for quite some time, so you need to manually generate the new `report-architecture.png` image.